### PR TITLE
Xray lasers now have unique sprites and inhands

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
imageadd: Xray lasers now have unique inhands.
imageadd: Xray lasers have been recolored to match R&D equipment.
/:cl:

technically a blob buff

### CODE REVIEW

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -75,7 +75,7 @@
 	name = "xray laser gun"
 	desc = "A high-power laser gun capable of expelling concentrated xray blasts that pass through multiple soft targets and heavier materials"
 	icon_state = "xray"
-	item_state = "laser"
+	item_state = null
 	origin_tech = "combat=5;materials=3;magnets=2;syndicate=2"
```
Comment (ChangelingRain) [inline code/modules/projectiles/guns/energy/laser.dm]: I would like to note here that xray lasers, which require combat 6, mat 5, bio 5, and power 4 to produce, give combat 5, mat 3, MAGNETS 2, and SYNDICATE 2.

Comment (Zilenan91) [inline code/modules/projectiles/guns/energy/laser.dm]: I thought Syndicate 2 on them got removed a long time ago. It should probably be gotten rid of.

Comment (ChangelingRain) [inline code/modules/projectiles/guns/energy/laser.dm]: oh well stuff for another PR i guess

Comment (Fox-McCloud) [inline code/modules/projectiles/guns/energy/laser.dm]: pretty sure it's intentional so there's *some* way of getting syndicate items besides it being a tator-tot round.

A lot of the augments require syndicate tech to get---as is it requires cargo or the experimentor to even make the X-Ray laserguns to begin with.

**Comment by WJohn** [comment]
:+1:

**Comment by KorPhaeron** [comment]
Close enough and I'm updating the server in a minute

File: code/modules/projectiles/guns/energy/special.dm [hunk 2]
```diff
@@ -79,6 +79,7 @@
 	name = "\improper Mind Flayer"
 	desc = "A prototype weapon recovered from the ruins of Research-Station Epsilon."
 	icon_state = "xray"
+	item_state = null
```
Comment (KorPhaeron) [inline code/modules/projectiles/guns/energy/special.dm]: Invisible?

Comment (KorPhaeron) [inline code/modules/projectiles/guns/energy/special.dm]: Joan assured me this works

Comment (KorPhaeron) [inline code/modules/projectiles/guns/energy/special.dm]: Is there not a less hacky way to do this?

Comment (ChangelingRain) [inline code/modules/projectiles/guns/energy/special.dm]: As I told kor, this is dumb but needed; https://github.com/tgstation/-tg-station/blob/master/code/modules/projectiles/guns/energy/nuclear.dm

**Pull request merged**